src/config/importVafConfig.nim

```diff
@@ -0,0 +1,24 @@
+import json
+import os
+import strutils
+
+import initVafConfig
+import parseVafConfig
+
+import VafConfigStructure
+
+var homeDir: string = os.getHomeDir()
```

Comment:
maybe put this in a separate file? it gets quite repetitive

Reply:
What should we put in a seperate file?

Reply:
> What should we put in a seperate file?

not sure, maybe a constsnts.nim in src/

Reply:
Good Idea! 👍 Lets refactor later tho